docs/index.asciidoc

```diff
@@ -424,6 +425,17 @@ When the `target` is set to a field reference, the `_source` of the hit is place
 This option can be useful to avoid populating unknown fields when a downstream schema such as ECS is enforced.
 It is also possible to target an entry in the event's metadata, which will be available during event processing but not exported to your outputs (e.g., `target \=> "[@metadata][_source]"`).
 
+
+[id="plugins-{type}s-{plugin}-retries"]
+===== `retries`
+
+* Value type is <<number,number>>
+* Default value is `0`
+
+The number of retries to run the query. If the query fails after all retries, it logs an error message.
```

Comment:
```suggestion
The number of times to re-run the query after the first failure. If the query fails after all retries, it logs an error message.
```

Reply:
I'm trying to make the description different from the option name to help clarify for anybody who might be confused.  Let me know what you think about this, and if you'd like to tweak it some more.